lib/common/common/DateUtilities.cpp

```diff
@@ -3,10 +3,8 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 #include "CommonCommonPch.h"
-#if _WIN32
 #define ENABLE_INTSAFE_SIGNED_FUNCTIONS 1
 #include <intsafe.h>
```

Comment:
> Do we still need to include intsafe.h here?  Would that make it not compile?
> May be we can remove the WinRTDate functions from ChakraCore as it is not used anywhere to avoid dependency to intsafe.h in this file as well?

It builds. Pal has intsafe.h (just missing Int64 functions). Currently the dependency in this file is HRESULT ```INTSAFE_E_ARITHMETIC_OVERFLOW``` returned by some functions.

Removing WinRTDate from ChakraCore seems a good idea. I'll look at that later.